modules/schema/src/main/scala/lucuma/odb/json/sourceprofile.scala

```diff
@@ -160,19 +219,17 @@ trait SourceProfileCodec {
 
 
   given (using Encoder[Wavelength]): Encoder[UnnormalizedSED] =
-    Encoder.instance { (sed: UnnormalizedSED) =>
-      sed match {
-        case StellarLibrary(librarySpectrum)          => Json.obj("stellarLibrary"  -> librarySpectrum.asJson)
-        case CoolStarModel(temperature)               => Json.obj("coolStar"        -> temperature.asJson) // todo: tag
-        case Galaxy(galaxySpectrum)                   => Json.obj("galaxy"          -> galaxySpectrum.asJson)
-        case Planet(planetSpectrum)                   => Json.obj("planet"          -> planetSpectrum.asJson)
-        case Quasar(quasarSpectrum)                   => Json.obj("quasar"          -> quasarSpectrum.asJson)
-        case HIIRegion(hiiRegionSpectrum)             => Json.obj("hiiRegion"       -> hiiRegionSpectrum.asJson)
-        case PlanetaryNebula(planetaryNebulaSpectrum) => Json.obj("planetaryNebula" -> planetaryNebulaSpectrum.asJson)
-        case PowerLaw(index)                          => Json.obj("powerLaw"        -> index.asJson)
-        case BlackBody(temperature)                   => Json.obj("blackBodyTempK"  -> temperature.value.value.asJson)
-        case UserDefined(fluxDensities)               => Json.obj("fluxDensities"   -> fluxDensities.toSortedMap.toList.map(EncoderFluxDensityEntry.apply).asJson)
-      }
+    Encoder.instance {
+      case StellarLibrary(librarySpectrum)          => Json.obj("stellarLibrary"  -> librarySpectrum.asJson)
+      case CoolStarModel(temperature)               => Json.obj("coolStar"        -> temperature.asJson) // todo: tag
```

Comment:
Did you intent to leave the comment?

Reply:
This was just reformatting to remove the explicit `match` statement.  I'm not sure what the `todo`, which was there before, refers to.